jax/_src/export/serialization.py

```diff
@@ -361,6 +364,10 @@ def _deserialize_pytreedef_to_pytree(p: ser_flatbuf.PyTreeDef):
     dtypes._float8_e4m3_dtype: ser_flatbuf.DType.f8_e4m3,
     dtypes._float8_e8m0fnu_dtype: ser_flatbuf.DType.f8_e8m0fnu,
     dtypes._float4_e2m1fn_dtype: ser_flatbuf.DType.f4_e2m1fn,
+
+    prng.KeyTy(prng.prngs["threefry2x32"]): ser_flatbuf.DType.key_fry,
+    prng.KeyTy(prng.prngs["rbg"]): ser_flatbuf.DType.key_rbg,
+    prng.KeyTy(prng.prngs["unsafe_rbg"]): ser_flatbuf.DType.key_unsafe_rbg,
```

Comment:
This reminds me that we may want to expose the key dtypes as public symbols, either via a function like `jr.key_dtype(...)` or individual symbols like `jr.threefry2x32_dtype`. Anyone can access them anyway e.g. by writing `jr.key(42, ...).dtype`.

No action to take in this PR. This is a note to self.

@jakevdp